static/js/hashchange.js

```diff
@@ -114,15 +117,6 @@ function do_hashchange_normal(from_reload) {
         case "#":
             activate_home_tab();
             break;
-        case "#keyboard-shortcuts":
-            info_overlay.show("keyboard-shortcuts");
-            break;
-        case "#message-formatting":
-            info_overlay.show("message-formatting");
-            break;
-        case "#search-operators":
-            info_overlay.show("search-operators");
-            break;
```

Comment:
I kept the `case` statements only here when merging since they provide some protection from future bugs.  (Just not the `info_overlay` lines -- i.e. these now do the same assert as `#drafts` (below))

Reply:
Okay!